cypress/commons/actions/brewery/BreweryParameters.js

```diff
@@ -128,6 +128,16 @@ const getCommentInput = () => ScenarioParameters.getParameterInput('text-input-c
 const getAdditionalDate = () => ScenarioParameters.getParameterValue('date-input-additional_date');
 const getAdditionalDateHelperText = () => cy.get('[id=date-text-field-additional_date-helper-text]');
 const getAdditionalDateInput = () => ScenarioParameters.getParameterInput('date-input-additional_date');
+const getScenarioToCompareInput = () => ScenarioParameters.getParameterInput('single-select-scenario_to_compare');
+const getScenarioToCompare = () => ScenarioParameters.getParameterValue('single-select-scenario_to_compare');
+
+const getScenarioToCompareSelect = () => {
+  return getScenarioToCompareInput();
+};
+const selectScenarioToCompareOption = (option) => {
+  getScenarioToCompareSelect().click();
+  cy.contains(option).click();
```

Comment:
Calling `cy.contains(option)` this way could lead to unexpected behavior, because it will return the first element in the DOM containing the `option` value.

Can we use the same mechanism with find/replace as the other select components instead ?

Example:
```js
function getCurrencySelectOption(option) {
  getCurrencySelect().click();
  cy.get(BREWERY_SELECTORS.scenario.parameters.basicTypes.currencyOption.replace('$OPTION', option)).click();
}
```